src/pages/expense/index.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { basicWrap } from 'styles/containers';
+import styled from '@emotion/styled';
+import { css } from '@emotion/react';
+import color from 'styles/colors';
+import Button, { ButtonType } from 'components/button';
+import TextInput from 'components/text-input';
+import { CaptionBold } from 'styles/typography';
+import Profile, { Animals } from 'components/profile';
+import UserCheckbox from './user-checkbox';
+
+const FormWrap = styled.div`
+  width: 100%;
+  min-height: 300px;
+  padding: 16px 16px 20px;
+  background: ${color.white};
+  border: 1px solid ${color.grayscale.gray05};
+  border-radius: 16px;
+  box-sizing: border-box;
+`;
+
+const Caption = styled(CaptionBold)`
+  color: ${color.grayscale.gray03};
+  margin-bottom: 12px;
+`;
+
+const PayerButton = styled.button`
+  width: 100%;
+  border: none;
+  background: none;
+  padding: 0;
+  margin-bottom: 21px;
+`;
+
+const SelectWrap = styled.div`
+  margin: 3px 5px;
+`;
+
+const MEMBERS = [
+  { id: 1, name: '지형', type: Animals.Rabbit, isMe: true },
+  { id: 2, name: '유진', type: Animals.Bear, isMe: false },
+  { id: 3, name: '주예', type: Animals.Unicorn, isMe: false },
+  { id: 4, name: '영진', type: Animals.Panda, isMe: false }
+];
+
+export default function Expense() {
+  return (
+    <div
+      css={[
+        basicWrap,
+        css`
+          background-color: ${color.grayscale.gray07};
+        `
+      ]}
+    >
+      <FormWrap>
+        <Button buttonType={ButtonType.Round} label="04.16" />
+        <div
+          css={css`
+            margin: 0 5px;
+          `}
+        >
+          <TextInput placeholder="금액입력(원)" type="number" />
+          <TextInput placeholder="내용입력" type="text" />
+        </div>
+        <SelectWrap>
+          <Caption>낸 사람</Caption>
+          <PayerButton>
+            <Profile name="지형" type={Animals.Rabbit} isMe hasName />
+          </PayerButton>
+
+          <Caption>쓴 사람</Caption>
+          {MEMBERS &&
+            MEMBERS.map &&
```

Comment:
`MEMBERS`가 배열인지 확인하는건가용??
```suggestion
            Array.isArray(MEMBERS) &&
```
`Array`객체가 아니더라도 `map`프로퍼티를 가질 수 있으니까 요렇게 하는건 어떨까영?

Reply:
건우님 리뷰 a5d13e9 커밋에 반영되었습니다~! •͈ᴗ•͈ 감사해용!

Reply:
73번째라인에 `MEMBERS && ...` 지우셔도 같은 결과입니당~!

Reply:
그렇겠네용! be0195d 해당 내용 반영해봤어요 ㅎㅎ